docs/faq/index.rst

```diff
@@ -11,25 +11,6 @@ Flocker is under active deployment and we receive a lot of questions about how t
 
```

Comment:
There's another...non sequitur?...above.

> Flocker is under active deployment and we receive a lot of questions about how this or that will be done in a future release.

It may not be "non sequitur"...but it doesn't make sense.

I think it (and the "get involved" line below) should be removed from the FAQ.